src/frontend/constants/api-constant.js

```diff
@@ -1,3 +1,6 @@
 export const API_LOGIN = "/api/auth/login";
 export const API_SIGNUP = "/api/auth/signup";
 export const API_USER_EDIT = "/api/users/edit";
+export const API_USERS = "/api/users/";
+export const API_FOLLOW = "/api/users/follow";
+export const API_UNFOLLOW = "/api/users/unfollow";
```

Comment:
export all these from one file ie. index.js

Reply:
Thanks for the review. These are API constants and hence exported form a single file.